### DISCUSSION TITLE
Better installation description for CLI tools.

### DISCUSSION THREAD

**Discussion by caycun** [rust-lang/crates.io#5887] [Feature Requests]
Make it so that if we are publishing a cli tool, we can edit the text in that install section that says :

`Run the following Cargo command in your project directory:
cargo add termongo`

change to `cargo install termongo` because it's a cli tool.

![image](https://user-images.githubusercontent.com/89197978/211190909-782148e2-3d9a-43b9-985b-a7e2e333d85f.png)

**Comment by Turbo87** [comment]
see https://github.com/rust-lang/crates.io/issues/5882 😉
